src/vm/i386/jitinterfacex86.cpp

```diff
@@ -1397,10 +1397,16 @@ void EmitFastGetSharedStaticBase(CPUSTUBLINKER *psl, CodeLabel *init, bool bCCto
         // DoInit:
         psl->EmitLabel(DoInit);
 
-#if defined(UNIX_X86_ABI)
-        // sub esp, 8 ; to align the stack
-        psl->X86EmitSubEsp(8);
-#endif
+#ifdef FEATURE_PAL
```

Comment:
It depends on how the unwinder works. E.g. it is not required on Windows because of Windows debuggers are happy to unwind it correctly as is.

I would either:
- Remove the `#ifdef FEATURE_PAL` and added the ebp frame everywhere. Even though it is not strictly required on Windows, it does not hurt to have it.
   or 
- Make the whole thing `UNIX_X86_ABI`